src/hmrc-content-guide/tariff-stop-press-notices/index.njk

```diff
@@ -14,6 +14,12 @@ layout: twoColumnPage.njk
 Use this pattern to tell users about changes to the UK Trade Tariff, Customs Handling of Import and Export Freight (CHIEF), Customs Declaration Service (CDS) and monthly trade euro rate.
 </p>
 
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">
+  Don't update a customs information paper after it's been published, unless it's to correct a typo. A new one must be issued.
```

Comment:
Please change 'Don't' to 'Do not'. Negative contractions are advised against in our style guide. We probably have this issue on other pages as just noticed on http://localhost:3000/hmrc-content-guide/customs-information-papers/ too.